config/locales/en.yml

```diff
@@ -64,7 +64,7 @@ en:
     error:
       generic: |
         Oops! Something went wrong and we couldn't process that action. You
-        might want to try it again, or if that doesn't work, refresh the page.
+        might want to try it again.
```

Comment:
The mobile app is essentially a web-view so technically we could refresh the page for the user but I am not sure about what to do with the message or if we should refresh without alerting the user?